spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -1143,5 +1137,46 @@ def test_get_interactive_backend(backend):
             assert value == '0'
 
 
+@flaky(max_runs=3)
+def test_debug_namespace(tmpdir):
```

Comment:
It seems this test doesn't work on Python 2. For simplicity, you can skip it because I doubt many people is still using it,

Reply:
I think Python2 test are generally broken. I guess pytest dropped compatibility with python 2